flexget/plugins/modify/sort_by_weighted.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+from datetime import datetime, timedelta
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+
+__author__ = 'andy'
+
+log = logging.getLogger('sort_by_weighted')
+
+SUPPORTED_TYPES = (
```

Comment:
The types? 
Mainly to safeguard from using unsupported fields. Sorting by string for example would need something like "similarity" between them for expected results.